scqc_reqs-1.0

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+PATH=/ei/software/cb/eipap/2.14.2/x86_64/bin:/ei/software/cb/eimap/2.1.4/x86_64/bin:/ei/software/cb/python_miniconda/4.9.2_py3.8_pap/x86_64/bin:/ei/software/cb/git/2.26.2_CBG/x86_64/bin:/ei/cb/common/Scripts:/tgac/software/production/bin/core/../..//hpccore/5/x86_64/bin:/usr/users/ga002/tgacpap/bin:/ei/software/cb/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/tgac/software/production/bin:/tgac/software/production/libraries/bin \
+source nextflow-22.04.0_CBG
```

Comment:
We do not need this file here. The PATH's are from PAP node, with eipap and eimap.

This should be part of the wrapper and sit under `/ei/software/cb` location.

```
cd /ei/software/cb/bin

cat > singlecellQC-1.0
#!/bin/bash
tool="singlecellQC/1.0"
location="/ei/software/cb"
echo "${tool} is sourced from ${location} location"

source nextflow-22.04.0_CBG

echo "Usage:"
echo "  nextflow run scqc_nf.sh -c config_file"

export PATH=/ei/software/cb/singlecellQC/1.0/x86_64/bin:$PATH
```

And `scqc_nf.sh` should be located under `/ei/software/cb/singlecellQC/1.0/x86_64/bin`

Reply:
changed.